documentation/0000-installing-nodejs.md

```diff
@@ -0,0 +1,17 @@
+- **Start Date:** (fill me in with today's date, YYYY-MM-DD)
```

Comment:
It would be helpful to have a start date!

Reply:
hey, youre the one who wrote thus!